daml-lf/language/daml-lf.bzl

```diff
@@ -88,17 +88,24 @@ LF_VERSIONS = [
     "1.12",
     "1.13",
     "1.14",
+    "1.15",
     "1.dev",
 ]
 
-# All LF versions for which we have protobufs.
-PROTO_LF_VERSIONS = [ver for ver in LF_VERSIONS if versions.gte(ver, "1.14")]
+# Next version of LF. not fully functional in general.
+lf_version_next = "1.15"
 
-# We support older LF versions using an older compiler binary
-LEGACY_COMPILER_LF_VERSIONS = [ver for ver in LF_VERSIONS if versions.lte(ver, "1.13")]
+# All LF versions for which we have protobufs.
```

Comment:
I'd consider defining this as a filter instead. We deliberately stopped listing some of these explicitly since we had bugs where we forgot to add a new version. With a filter on top of the existing list, this is slightly more robust.